### DISCUSSION TITLE
Truecharts Custom App

### DISCUSSION THREAD

**Discussion by KickingAss2024** [MDeLuise/plant-it#158] [Q&A]
As there is no app at the moment for truecharts has any one else installed it with the Truecharts Custom App.

If so whatvis your configuration?

**Comment by MDeLuise** [comment]
Hi @adtwomey,

Thanks for reaching out. The charts for Plant-it [are now available within TrueCharts](https://truecharts.org/charts/incubator/plant-it/).

Please note that I am not the creator of these charts. For any issues or detailed instructions on how to deploy using TrueCharts, please refer to the [official TrueCharts documentation](https://truecharts.org/guides/).

Also, I wanted to mention that I create Helm charts for the app, and you can find instructions on how to use them [here](https://docs.plant-it.org/0.7.1/server-installation/#kubernetes-deployment). So besides the charts provided by TrueCharts, you also have the option to use directly provided charts.
